### PR TITLE
Rate dynamic sliders

### PR DESCRIPTION
**Pull request by AndreySemjonov**
New look of Rate dynamics 
image

### CODE REVIEW

**Comment by nerdCopter** [comment]
help icons?

**Comment by AndreySemjonov** [comment]
> ~help icons?~
> added help-icons.
> 

sorry, forgot.

**Comment by nerdCopter** [comment]
compiled-install looks different than debug-run and i did not modify such

**Comment by nerdCopter** [comment]
* difficult to set to html defaults, easier to set some new CSS.
* should be fair/acceptable for now.

**Comment by nerdCopter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5576260&bid=18281391)

File: src/css/main.css [hunk 1]
```diff
@@ -53,6 +53,41 @@ a.disabled {
...
 }
 
+/* The slider (use -webkit- (Chrome, Opera, Safari, Edge) and -moz- (Firefox) and -ms- (IExplorer) to override default look) */
+input[type=range] {
+  -webkit-appearance: none;
+  /*height: 30px;*/
+  /*margin: 10px 0;*/
+  width: 100%;
+}
+input[type=range]:focus {
```
Comment (nerdCopter) [inline src/css/main.css:63]: Codacy Issue found: [Outlines shouldn't be hidden unless other visual changes are made.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5576260)

**Comment by nerdCopter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5576260&bid=18281391)

File: src/css/main.css [hunk 2]
```diff
@@ -53,6 +53,41 @@ a.disabled {
...
 }
 
+/* The slider (use -webkit- (Chrome, Opera, Safari, Edge) and -moz- (Firefox) and -ms- (IExplorer) to override default look) */
+input[type=range] {
+  -webkit-appearance: none;
+  /*height: 30px;*/
+  margin: 0px 1;
+  width: 100%;
+}
+input[type=range]:focus {
```
Comment (nerdCopter) [inline src/css/main.css:63]: Codacy Issue found: [Outlines shouldn't be hidden unless other visual changes are made.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5576260)

**Comment by nerdCopter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5576260&bid=18281391)

File: src/css/main.css [hunk 3]
```diff
@@ -53,6 +53,42 @@ a.disabled {
...
 
+/* The slider (use -webkit- (Chrome, Opera, Safari, Edge) and -moz- (Firefox) and -ms- (IExplorer) to override default look) */
+input[type=range] {
+  -webkit-appearance: none;
+  /*height: 30px;*/
+  /*margin: 10px 0;*/
+  width: 100%;
+}
+input[type=range]:focus {
+  border: 1px solid 2497E3;
```
Comment (nerdCopter) [inline src/css/main.css:64]: Codacy Issue found: [Expected RBRACE at line 64, col 21.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5576260)

**Pull request merged**